app/assets/javascripts/oxalis/model/volumetracing/volumelayer.js

```diff
@@ -46,11 +46,15 @@ export class VoxelIterator {
     this.minCoord2d = minCoord2d;
     this.get3DCoordinate = get3DCoordinate;
     this.boundingBox = boundingBox;
-    const firstCoordinate = this.get3DCoordinate(this.minCoord2d);
-    if (this.map[0][0] && this.isCoordinateInBounds(firstCoordinate)) {
-      this.next = firstCoordinate;
+    if (!this.map || !this.map[0]) {
+      this.hasNext = false;
```

Comment:
Can we add a `return` statement after this line and hoist the following else-block one level up?